<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentTask</class>
 <widget class="QDialog" name="ContentTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 199, 167);</string>
  </property>
  <widget class="QLineEdit" name="lineTitulo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="datePrazo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_prioridade">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
   <item>
    <property name="text">
     <string>Urgente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Crítico</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textCorpo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
   <property name="text">
    <string>Prioridade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
   <property name="text">
    <string>Titulo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
   <property name="text">
    <string>Corpo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
   <property name="text">
    <string>Prazo:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConcluir">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Concluir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Descreva sua tarefa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_type">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black ;</string>
   </property>
   <item>
    <property name="text">
     <string>Pessoal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Profissional</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Acadêmico</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
